PeraWalletCore/Models/Settings/SecretDeveloperSettings.swift

```diff
@@ -14,19 +14,26 @@
 
 //   SecretDeveloperSettings.swift
 
-enum SecretDeveloperSettings {
-    
+import UIKit
+import pera_wallet_core
+
+public enum SecretDeveloperSettings: Settings, Hashable {
     case enableTestCards
+    case enableTestXOSwapPage
     
-    var image: ImageResource {
+    public var image: UIImage? {
         switch self {
         case .enableTestCards: .iconMenuCards
+        case .enableTestXOSwapPage: img("tabbar-icon-swap-selected")
         }
     }
     
-    var name: String {
+    public var name: String {
         switch self {
         case .enableTestCards: String(localized: "secret-dev-settings-enable-test-cards")
+        case .enableTestXOSwapPage: String(localized: "secret-dev-settings-enable-test-xoswap")
         }
     }
+
+    public var subtitle: String? { nil }
```

Comment:
I think this revert my recent changes. Are you sure that we need `public` here if you import `pera_wallet_core` into this file?